source/includes/_events.md

```diff
@@ -1,5 +1,27 @@
 # Events
 
+An Event Resource is a top-level resource that represents a call to action that has a location.
+
+It contains the following attributes.
+
+Name    | Type | Description
+--------- |  ----------- |  -----------
+identifiers | array | A unique string array of identifiers in the format [system name]:[id]. See the general concepts document for more information about identifiers.
```

Comment:
"General concepts document" needs to be removed or changed since this isn't the OSDI docs, it's our copy-paste of it